citations.md

```diff
@@ -3,6 +3,7 @@
 ### 2020
 
 * Glynatsi, Nikoleta E., and Vincent A. Knight. "A meta analysis of tournaments and an evaluation of performance in the Iterated Prisoner's Dilemma" arXiv preprint arXiv:2001.05911 (2019). [ArXiv](https://arxiv.org/abs/2001.05911)
+* Wang, Shiheng, and Fangzhen Lin. "Nice Invincible Strategy for the Average-Payoff IPD." https://www.aaai.org/Papers/AAAI/2020GB/AAAI-WangS.1218.pdf
```

Comment:
Can we make this into a link, [pdf] if nothing else?

Reply:
Good idea :+1:

Reply:
d826d43